controllers/control_plane_init.go

```diff
@@ -0,0 +1,48 @@
+package controllers
+
+import (
+	"encoding/json"
```

Comment:
sigs.k8s.io/yaml?

Reply:
Maybe. Depending on if we need that much power. Right now it should be sufficient to generate the configurations as JSON docs, but we may end up needing yaml at some point.